lib/grape/dsl/request_response.rb

```diff
@@ -104,7 +104,7 @@ def rescue_from(*args, &block)
             handler = block
           end
 
-          options = args.extract_options!
+          options = args.extract_options!.dup
```

Comment:
That's not what I meant. This particular one actually returns a new hash every time according to the documentation, so a `dup` is just doing extra unnecessary work.